org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/parser/CompUtil.java

```diff
@@ -52,6 +54,8 @@
 /**
  * This class provides convenience methods for calling the parser and the
  * compiler.
+ *
+ * @modified Nuno Macedo // [HASLab] electrum-base
```

Comment:
adds a utility methods to tests whether a command is temporal (used to decide whether to employ temporal analysis and visualisation)